test/datasets.py

```diff
@@ -3,7 +3,7 @@
 
 def retrieve_dataset(dataset, **kwargs):
     # if data not extracted, download zip and extract
-    outdirname = 'datasets.7.20.2020'
+    outdirname = 'datasets.10.29.2020'
```

Comment:
Is this required? Might be good to remove the dependency on any one date?

Reply:
I created a new zip folder with the new dataset in blob storage, so it is required